package.json

```diff
@@ -22,7 +22,7 @@
     "build": "lerna run build:prod --stream",
     "fetch-config": "lerna run --parallel --stream fetch-config --",
     "lint": "lerna run lint --parallel --stream",
-    "lint:fix": "lerna run lint:fix --parallel --stream --since master --include-dependencies",
+    "lint:fix": "lerna run lint:fix --parallel --stream --since master --include-dependencies --",
```

Comment:
Need `--` at the end to fix this
![Annotation 2020-06-19 081646](https://user-images.githubusercontent.com/26746668/85087023-7d4d7e80-b206-11ea-96c2-6005503809b1.png)

Reply:
I think can revert this